src/main/scala/Syntax.scala

```diff
@@ -1,11 +1,19 @@
 package fuselang
 
 import scala.util.parsing.input.Positional
+import scribe._
 
 object Syntax {
 
   import Errors._
 
+  /** Makes all positionals logable by scribe */
+  implicit object PositionalLoggable extends Loggable[(String,Positional)] {
+    override def apply(value: (String, Positional)) = {
+      new output.TextOutput(s"${value._1}\n${value._2.pos.longString}")
+    }
+  }
```

Comment:
How is this used? Do I simply do `logger.warn(expr)`? Can you write an example (in the github thread) of how you use it?